locales/en.json

```
@@ -213,6 +219,15 @@
   "Settings_MagicPlus_Enable": "Enable Magic+",
   "Settings_MagicPlus_Description_Optimization": "Automatically optimize your task organization to improve your productivity",
   "Settings_MagicPlus_ExplanationLocal": "Fully local model, no data is transferred outside your device",
+  "Settings_MagicPlus_Unloading": "Unloading of Magic+",
```

```suggestion
  "Settings_MagicPlus_Unloading": "Disabling Magic+",
```
même changement en anglais :)